torch/autograd/gradcheck.py

```diff
@@ -485,12 +488,11 @@ def get_numerical_jacobian_wrt_specific_input(
 def _get_analytical_jacobian_forward_ad(
     fn, inputs, outputs, *, check_grad_dtypes=False, all_u=None
 ) -> Tuple[Tuple[torch.Tensor, ...], ...]:
-    """Computes the analytical Jacobian using forward mode AD of `fn(inputs)` using forward mode AD with respect
-    to `target`. Returns N * M Jacobians where N is the number of tensors in target that require grad and
-    M is the number of non-integral outputs.
-    Contrary to other functions here, this function requires "inputs" to actually be used by the function.
-    The computed value is expected to be wrong if the function captures the inputs by side effect instead of
-    using the passed ones (many torch.nn tests do this).
+    """Calculate analytical Jacobian using forward mode AD for `fn(inputs)` w.r.t. `target`, with N * M Jacobians.
```

Comment:
Can we apply fewer modifications to this docstring and only stick to fixing pydocstyle errors.

Reply:
I have modified one-liner. please check this.